src/components/NewTodo/NewTodo.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './NewTodo.css';
+
+export class NewTodo extends React.Component {
+  state = {
+    users: this.props.users,
+    title: '',
+    userId: 0,
+    id: 3,
+    errorTitle: false,
+    errorUser: false,
+  }
+
+  onSubmit = (e) => {
+    e.preventDefault();
+
+    const { userId, id, title } = this.state;
+
+    this.setState((prevState) => {
+      if (title.length === 0) {
+        return {
+          errorTitle: true,
+        };
+      }
+
+      if (userId === 0) {
+        return {
+          errorUser: true,
+        };
+      }
+
+      const todo = {
+        userId, id, title,
+      };
+
+      this.props.addTodo(todo);
```

Comment:
don't call addTodo in setState

Reply:
make a separate validation function and return some object with errors or just one value for example and then check it in this method and return setState with errors on addTodo in `else` statement